app/views/episode_media/_form.html.erb

```diff
@@ -14,9 +14,7 @@
           </div>
         </div>
         <div class="col-lg-4 d-grid align-content-start gap-3">
-          <div class="sticky-lg-top">
-            <%= render "form_status", episode: episode, form: form %>
-          </div>
```

Comment:
Since this is the only page that uses off canvas, if we remove the sticky (which isn't needed), it solves our z-index charade here.